### DISCUSSION TITLE
Unknown module 'if_mib'

### DISCUSSION THREAD

**Discussion by bcheevers123** [prometheus/snmp_exporter#874] [Q&A] [answered]
Hello. I am running snmp_exporter on my RPI 4 which runs buster raspbian 32 bit.

I get an Unknown module 'if_mib' error hen I navigate to the 127.0.0.1:9116 address.

This is strange as the IF-MIB.txt is in the miblist.txt file in /usr/share/snmp/mibs

I seem to not be able to make the generator for the snmp.yml file, as it gets perpetually stuck on the go:finding stage

I am following this tutorial, https://www.youtube.com/watch?v=LDFalgu3Vtg&ab_channel=RobustPerception which unfortunatly does not go into detail regarding this.

Could someone make a snmp.yml file for me? I simply require as much networking information from my Windows hosts. Or perhaps help me resolve the issues?

**Comment by SuperQ** [comment, marked as answer]
There is an example snmp.yml in this repo.
